### PR TITLE
f5

### PR DESCRIPTION
**Pull request by ShahidManzoor5**

### CODE REVIEW

**Comment by ShahidManzoor5** [comment]
commit

File: Server/API-Endpoints.txt [hunk 1]
```diff
@@ -44,12 +44,25 @@
...
 
+7. Verfy email
+ url: http://localhost:8000/auth/user/verify-email
+ type:GET
+ body:{
+  NULL
+ }
+
+8. Resend Email verfication
+  url: http://localhost:8000/auth/user/resend-verification-email
```
Comment (DevFaisal) [inline Server/API-Endpoints.txt:61]: This is in now in separate controller

**Review by developer-faisal — APPROVED** [review]
Add all API points

**Comment by ShahidManzoor5** [comment]
F7

**Pull request closed without merging**